testapps/SimpleAppWithAssemblyRename/Views/Home/About.cshtml

```diff
@@ -0,0 +1,7 @@
+﻿@{
```

Comment:
We can simplify this app - just have an Index page sans Layout \ _ViewImport, _ViewStart.